### PR TITLE
Add untz command

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -386,7 +392,15 @@
...
     }, fix);
     return 0;
- }
+  },
+
+  untz : function(obj) {
+    untz = true;
+  },
+
+  nountz : function(obj) {
```
Comment (bifurcation) [inline index.html]: Would be a little nicer to do this as `/untz start` and `/untz stop`, like with the sounds.

Comment (bifurcation) [inline index.html]: Also, comments?

**Comment by martinthomson** [comment]
LGTM, rebase

**Comment by ekr** [comment]
This time for sure.

**Pull request merged**